docs/developer/conventions/test/online.rst

```diff
@@ -95,20 +95,124 @@ Example fixture ~/.geotools/postgis/local.properties::
 In windows you cannot create a ".geotools" folder!
 
 1. Open up a CMD window
-2. cd to your home directory and use mkdir
-    
-    C:\Documents and Settings\Fred>mkdir .geotools
-
-3. And set up any fixtures you need:
-    
-    C:\Documents and Settings\Fred>cd .geotools
-    C:\Documents and Settings\Fred\.geotools>mkdir postgis
-    C:\Documents and Settings\Fred\.geotools>cd postgis
-    C:\Documents and Settings\Fred\.geotools\postgis>notepad typical.properties
-
-4. And use the as a guide: https://github.com/geotools/geotools/tree/master/build/fixtures
+2. cd to your home directory and use mkdir::
+
+    C:\\Documents and Settings\\Fred\>mkdir .geotools
+
+
+3. And set up any fixtures you need::
+
+    C:\\Documents and Settings\\Fred>cd .geotools
+    C:\\Documents and Settings\\Fred\\.geotools>mkdir postgis
+    C:\\Documents and Settings\\Fred\\.geotools>cd postgis
+    C:\\Documents and Settings\\Fred\\.geotools\\postgis>notepad typical.properties
```

Comment:
Many test cases write out a template in `~/.geotools/` so you have something to edit.

Reply:
this is just a formatting fix in the existing doc as you can see in  https://docs.geotools.org/latest/developer/conventions/test/online.html it renders as shown below

![image](https://user-images.githubusercontent.com/1165786/82021523-43c7a780-968b-11ea-98dc-1b9f36d5b2d8.png)